README.md

```diff
@@ -16,12 +16,16 @@ This repo contains the sources of the app, netlify functions and configuration f
     
 ### 3 Dev Server :
 
-* To test your code you can run:
+* To test your code you can run :
```

Comment:
In facts, there is no space between the text and the two points in English (cf. http://www.la-ponctuation.com/deux-points.html).

Reply:
Indeed! I'll change it